src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -664,12 +669,13 @@ impl HardcodedLayers {
              , &above_nodes_text
              , &panel
              , &panel_text
+             , &node_searcher
```

Comment:
Add comment in code, why we don't keep this list in one line (so nobody will ever try to do this).